### ISSUE TITLE
Is there way to find the client(initiator who started the conversation) request in the netflow data ?

### ISSUE THREAD

**Issue by Krishjs** [pmacct/pmacct#353]
Hi,

Is there way to find the client(initiator who started the conversation) request in the netflow data ?
Is there way to apply aggregate filter to get only the initial request ?

**Comment by paololucente** [comment]
Hi @Krishjs ,

The potential danger is that in NetFlow/IPFIX, by design, tcpflags are ORed as they come through and the flow is cached. Plus, of course, if you have any sampling in place you can simply miss TCP flags. This said, you can try to isolate SYNs with an aggregate_filter just like you asked in #350 and probably pair that with an aggregate a-la 'peer_src_ip, src_host, dst_host, proto, tos, src_port, dst_port, tcpflags'.

In short YMMV with this using NetFlow/IPFIX as input.

Paolo
